tests/unit/test_export_to_nitrate.py

```diff
@@ -48,7 +48,7 @@ def test_export_to_nitrate_expect(self):
 
         expect = convert_manual_to_nitrate(file_name)[1]
         html_generated = """<b>Test</b>\
-<p>Step 1.</p><p>Text similar to the one bellow is displayed
+<p>Step 1.</p><p>Text similar to the one below is displayed
```

Comment:
:thinking:

Reply:
Seems correct. Or is there something more to it than just "below" vs "bellow"?